packages/gafl-webapp-service/src/pages/terms-and-conditions/__tests__/route.spec.js

```diff
@@ -13,4 +11,108 @@ describe('terms-and-conditions > route', () => {
       expect(pageRoute).toBeCalledWith('terms-and-conditions', '/buy/conditions', validator, nextPage, getData)
     })
   })
+
+  describe('getData', () => {
+    const mockRequest = (statusGet = () => {}, transactionGet = () => {}) => ({
```

Comment:
Convention is `getMockRequest` or `generateMockRequest`, if you wouldn't mind renaming?